package/contents/ui/lib/WindowLayout.qml

```diff
@@ -123,9 +177,14 @@ PlasmaComponents.Button {
                     // Register shortcuts
                     if (window.shortcutKey) {
                         let key = [window.shortcutModifier, window.shortcutKey];
-                        tileShortcuts.set(key, () => {
-                            tileWindow(activeClient, window.x, window.y, window.width, window.height);
-                        });
+                        main.tileShortcuts.set(key, function(workspace, window, tileWindow, root) {
+                            return function() {
+                                if (window == undefined || root == undefined || workspace == undefined || workspace.activeClient == undefined) {
+                                    return;
+                                }
+                                tileWindow(workspace.activeClient, window, root);
+                            }
+                        }(workspace, window, tileWindow, root));
```

Comment:
I'm not sure I understand what you did here. Would be great if you can explain. And those checks would be better suited inside `tileWindow`

Reply:
Also just tested and shortcuts don't seem to work at all.

Reply:
That's _really_ odd.  This is while the dialog is open, correct?

It's definitely working inside my Debian12 VM.

(It made life easier.  I can mount the filesystem with sshfs to easily copy over the edited files, ssh over to watch the log output, and logout and log back in every time I need to test a change.)

With that said, if you add a console.log statement just before the tileWindow call at what is currently line 185, and one at the start of the function at line 182, do either show up in the logs?